signingscript/test/test_task.py

```diff
@@ -49,3 +51,46 @@ def test_missing_mandatory_urls_are_reported(context):
 def test_no_error_is_reported_when_no_missing_url(context):
     context.task = task_generator.generate_object()
     validate_task_schema(context)
+
+
+@pytest.mark.asyncio
+async def test_execute_post_signing_steps(context, monkeypatch):
+    work_dir = context.config['work_dir']
+    abs_to = os.path.join(work_dir, 'target.apk')
+
+    async def zip_align_apk_mock(_context, _abs_to):
+        assert context == _context
+        assert abs_to == _abs_to
+
+    def get_hash_mock(_abs_to, hash_type):
+        assert abs_to == _abs_to
+        assert hash_type in ('sha512', 'sha1')
+
+    monkeypatch.setattr('signingscript.task._zip_align_apk', zip_align_apk_mock)
```

Comment:
TIL monkeypatch.  I've been using mock / mocker, which seems to be similar.  Both work :)